modules/ROOT/pages/osv2-connected-apps.adoc

```diff
@@ -0,0 +1,72 @@
+= Configuring Connected Apps
+
+The Connected Apps feature enables you to integrate external applications with Anypoint Platform by providing access to those applications without sharing your user credentials.
+
+For information about Connected Apps, see 
+xref:access-management::connected-apps-overview.adoc[] and
+xref:access-management::connected-apps-developers.adoc[].
+
+
+[[configure-os-conn-app]]
+== Configure a Connected App in Access Management
+
+To create a Connected App with object store scopes:
+
+. Log in to Anypoint Platform and select *Access Management*.
+. In the Access Management navigation menu, click *Connected Apps*.
+. Click *Create App*.
+.. Specify a name for the app.
+.. Select *App acts on its own behalf (client credentials)*.
+.. Click *Add Scopes*.
+.. Select the object store scopes to apply:
++
+image::os-conn-apps-scopes.png[Scopes for Object Store connected apps]
+.. Click *Next* and select the business groups and environments to apply the scopes to.
+.. Click *Review*.
+.. Click *Add scopes* or *Cancel* to cancel your changes.
+. Click *Save* or *Cancel* to cancel your changes.
+. From the *Connected Apps* page, click *Copy Id* and *Copy Secret* for the app you just created.
++
+Use this client ID and client secret to get the access token required to authenticate with the Object Store v2 API.
++
+For information on getting the access token for Connected Apps, see
+xref:access-management::connected-apps-developers.adoc#oidc-endpoints-and-tokens[OIDC Endpoints and Tokens].
+
+[[configure-mq-conn-app-api]]
+== Configure a Connected App Using the API
+
+For information on configuring a Connected App with the API, see
+https://help.mulesoft.com/s/article/How-to-create-connected-apps-creation-using-API[How to create connected apps using the API].
+
+[[osv2-scopes]]
+=== Object Store Scopes
+
+Object store scopes apply to all stores in the given organization and environment.
+Unless otherwise noted, these scopes apply to both object stores and Object Store v2 object stores.
+
+[%header,cols="20,20,60a"]
+|===
+|Scope in UI|Scope in API |Description
+|*Manage stores*
+|`manage:store`
+|Create, read, update, and delete stores.
+|*Manage stores data*
+|`manage:store_data`
+|Perform all store operations including data, partition, and confirmation APIs. 
+|*View stores*
+|`read:store`
+|Read store details.
+|*Manage store clients* (object store only)
```

Comment:
Can we add N/A for scopes in API for scopes which do not apply to OSv2 instead of leaving it blank?